dm/pkg/conn/mockdb.go

```diff
@@ -61,3 +73,106 @@ func InitVersionDB(c *check.C) sqlmock.Sqlmock {
 	}
 	return mock
 }
+
+// TODO: export Config in https://github.com/pingcap/tidb/blob/a8fa29b56d633b1ec843e21cb89131dd4fd601db/br/pkg/mock/mock_cluster.go#L35
```

Comment:
why we need copy cluster code here? seems only difference is `Port`?

Reply:
port is not export in tidb repo, so I copy the main logic now. We may change back after 5.4 release or maintain a mock cluster in DM for further use.

Reply:
No, after https://github.com/pingcap/tidb/pull/31167, mock cluster use a random port but not export in DSN(127.0.0.1:/)